tools/matchms/matchms_filtering.xml

```diff
@@ -0,0 +1,153 @@
+<tool id="matchms_filtering" name="matchMS filtering" version="0.11.0+galaxy0">
+    <description>calculate the similarity score and matched peaks</description>
+    <requirements>
+        <requirement type="package" version="0.11.0">matchms</requirement>
+        <requirement type="package" version="1.1.4">pandas</requirement>
+    </requirements>
+
+    <environment_variables>
+        <environment_variable name="MPLCONFIGDIR">\$_GALAXY_JOB_TMP_DIR</environment_variable>
+    </environment_variables>
+
+    <command detect_errors="exit_code"><![CDATA[
+        sh ${matchms_python_cli}
+    ]]> </command>
+
+    <configfiles>
+        <configfile name="matchms_python_cli">
+            python3 ${__tool_directory__}/matchms_filtering_wrapper.py \
+            --spectra "$spectra" \
+            --spectra_format "$spectra.ext" \
+            #if $normalise_intensities.is_true
+                -normalise_intensities \
+            #end if
+            #if $default_filters.is_true
+                -default_filters \
+            #end if
+            #if $clean_metadata.is_true
+                -clean_metadata \
+            #end if
+            #if $relative_intensity.is_true
+                -relative_intensity \
+                --from_intensity "$relative_intensity.from_intensity" \
+                --to_intensity "$relative_intensity.to_intensity" \
+            #end if
+            #if $mz_range.is_true
+                -mz_range \
+                --from_mz "$mz_range.from_mz" \
+                --to_mz "$mz_range.to_mz" \
+            #end if
+            --output "$output"
+        </configfile>
+    </configfiles>
+
+    <inputs>
+        <param label="Spectra file" name="spectra" type="data" format="msp,mgf" help="Mass spectra file to be filtered." />
+
+        <conditional name="normalise_intensities">
+            <param name="is_true" label="Normalize intensities" type="boolean" truevalue="TRUE" falsevalue="FALSE" checked="false"
+                   help="Normalize intensities of peaks (and losses) to unit height."/>
+        </conditional>
+
+        <conditional name="default_filters">
+            <param name="is_true" label="Apply default filters" type="boolean" truevalue="TRUE" falsevalue="FALSE" checked="false"
+                   help="Collection of filters that are considered default and that do no require any (factory) arguments."/>
+        </conditional>
+
+        <conditional name="clean_metadata">
+            <param name="is_true" label="Clean metadata" type="boolean" truevalue="TRUE" falsevalue="FALSE" checked="false"
+                   help="Apply all adding and cleaning filters if possible, so that the spectra have canonical metadata." />
+        </conditional>
```

Comment:
I am not sure the first three checkboxes need to have a conditional. There is nothing on the toolform being triggered by the selection, could these be just checkboxes?

Reply:
Yes, you are right, the reason is purely aesthetic so that all the filters looks the same. Also, to the best of my knowledge, you can't add a `help` section to a checkbox option.